CHANGELOG.md

```diff
@@ -7,4 +7,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Example use case of spec-generated Ruby client. 
-[cybearl/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
+[cybeark/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
+- Added the /whoami endpoint to the spec file
```

Comment:
```suggestion
- The /whoami endpoint is now included in the spec.
```

Please make sure, following our [changelog guidelines](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-guidelines) that the entries are sentences that end in periods and don't repeat the section headers (eg no need to say "Added" again)

We also try to be clear about user impact - it might be nice to say a little about why each endpoint we're documenting is useful in these entries